edgelet/edgelet-http-mgmt/src/server/module/mod.rs

```diff
@@ -55,7 +56,16 @@ where
         Err(err) => return Err(Error::from(err.context(context))),
     };
 
-    let module_spec = match CoreModuleSpec::new(name, type_, config, env) {
+    let pull_policy = match spec
+        .pull_policy()
+        .map_or(Ok(PullPolicy::default()), |policy| {
+            PullPolicy::from_str(policy)
```

Comment:
Use `str::parse` instead of `FromStr::from_str`, ie `policy.parse()`

(Clippy will then probably tell you to use UFCS, ie `.map_or(Ok(PullPolicy::default()), str::parse)`)

Reply:
I see, sure will do!